appium/tests/specs/composeEmail/SentEncryptedEmailAfterWrongPassPhrase.spec.ts

```diff
@@ -0,0 +1,51 @@
+import {
+  SplashScreen,
+  CreateKeyScreen,
+  InboxScreen,
+  NewMessageScreen,
+  EmailScreen,
+  MenuBarScreen
+} from '../../screenobjects/all-screens';
+
+import { CommonData } from '../../data';
+import DataHelper from "../../helpers/DataHelper";
+
+describe('COMPOSE EMAIL: ', () => {
+
+  it('user is able to sent encrypted email after reseting pass phrase', async () => {
+
+    const contactEmail = CommonData.secondContact.email;
+    const emailSubject = CommonData.simpleEmail.subject + DataHelper.uniqueValue();
+    const emailText = CommonData.simpleEmail.message;
+    const passPhrase = CommonData.account.passPhrase;
+    const wrongPassPhraseError = CommonData.errors.wrongPassPhrase;
+    const wrongPassPhrase = "wrong";
+    const senderEmail = CommonData.account.email;
+
+
+    await SplashScreen.login();
+    await CreateKeyScreen.setPassPhrase();
```

Comment:
`CreateKeyScreen` should be called `SetupKeysScreen` since no key is being created. Filing another issue.